Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM jboss/base-jdk:8
+
+MAINTAINER Andrew Lee Rubinger (https://github.com/ALRubinger)
+
+# install wget
+USER root
+
+RUN yum install -y wget
```

Comment:
please run yum in a way of https://github.com/jboss-dockerfiles/base-jdk/blob/jdk7/Dockerfile#L8 - to preserve space